### PR TITLE
Bug slayer template

### PR DESCRIPTION
**Pull request by xcorail**
Create a form for the revisited Bug Slayer bounty

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-slayer.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        - [CVE-20nn-xxxx](<relevant URL>)
+        - [CVE-20nn-yyyy](<relevant URL>)
+    validations:
+      required: true
+  - type: input
+    id: a41_url 
+    attributes:
+      label: All For One submission
+      description: Link to the All For One submission with your CodeQL query
+      placeholder: ex. https://github.com/github/codeql/pull/nnnn
```
Comment (pwntester) [inline .github/ISSUE_TEMPLATE/bug-slayer.yml:34]:
```suggestion
      placeholder: |
        ex. https://github.com/github/codeql/pull/nnnn
```

Comment (pwntester) [reply .github/ISSUE_TEMPLATE/bug-slayer.yml:34]: Not sure, if the colon is causing the viewing issue (the same one Im getting for the A41 one) so moving it to a string literal (double quotes should also work)

**Review by pwntester — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug-slayer.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      placeholder: |
+        ex.
+        - [CVE-20nn-xxxx](<relevant URL>)
+        - [CVE-20nn-yyyy](<relevant URL>)
+    validations:
+      required: true
+  - type: input
+    id: a41_url 
+    attributes:
+      label: All For One submission
```
Comment (intrigus-lgtm) [inline .github/ISSUE_TEMPLATE/bug-slayer.yml:32]: Why is a (successful?) all-for-one application now required for the bug slayer?
I don't think this was required previously?
Unless I interpreted the rules wrong^^

**Comment by pwntester** [comment]
fixed a couple of things to make it render, should be ok now

File: .github/ISSUE_TEMPLATE/bug-slayer.yml (hunk 2 above)
Comment (xcorail) [reply .github/ISSUE_TEMPLATE/bug-slayer.yml:32]: @intrigus-lgtm we are revisiting the Bug Slayer
More news coming shortly

File: .github/ISSUE_TEMPLATE/bug-slayer.yml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+description: Submit a CodeQL query for the Bug Slayer bounty (https://securitylab.github.com/bounties)
+title: "[<language>]: <short description>"
+labels: [The Bug Slayer]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        # Introduction
+
+        Thank you for submitting a query to the GitHub CodeQL project!
```
Comment (xcorail) [inline .github/ISSUE_TEMPLATE/bug-slayer.yml:11]:
```suggestion
        Thank you for your submission to the bounty program!
```

File: .github/ISSUE_TEMPLATE/bug-slayer.yml [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+        - [CVE-20nn-yyyy](<relevant URL>)
+    validations:
+      required: true
+  - type: input
+    id: a41_url 
+    attributes:
+      label: All For One submission
+      description: Link to the All For One submission with your CodeQL query
+      placeholder: |
+        ex. https://github.com/github/codeql/pull/nnnn
```
Comment (xcorail) [inline .github/ISSUE_TEMPLATE/bug-slayer.yml:35]:
```suggestion
        ex. https://github.com/github/securitylab-bounties/issues/nnn
```

File: .github/ISSUE_TEMPLATE/bug-slayer.yml [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+      label: Original author
+      description: Your GitHub handle or the GitHub handle of the original author of the query
+      placeholder: ex. @myhandle
+    validations:
+      required: true
+  - type: input 
+    id: splitscheme
+    attributes:
+      label: Split scheme
+      description: If you are not the original author, split scheme for the bounty
```
Comment (xcorail) [inline .github/ISSUE_TEMPLATE/bug-slayer.yml:50]:
```suggestion
      description: If you are not the original author, split scheme for the bounty, under the form "original_author/you"
```

File: .github/ISSUE_TEMPLATE/bug-slayer.yml [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+    validations:
+      required: true
+  - type: input 
+    id: splitscheme
+    attributes:
+      label: Split scheme
+      description: If you are not the original author, split scheme for the bounty, under the form original_author/yourself. For example 30/70 means that 30% of the reward will go to the original author, and 70% to you. 
+      placeholder: ex. 50/50
+    validations:
+      required: false
```
Comment (xcorail) [inline .github/ISSUE_TEMPLATE/bug-slayer.yml:53]:
```suggestion
```

File: .github/ISSUE_TEMPLATE/bug-slayer.yml [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+        After you submit this issue, the GitHub Security Lab and CodeQL teams will triage the submission and, if it meets the Query Bounty Program requirements, we will grant you a bounty through our HackerOne program.
+
+        Please make sure to carefully read the [bounty program description and conditions](https://securitylab.github.com/bounties/)
+
+        # Questionnaire
+  - type: textarea
+    id: cve_ids
+    attributes:
+      label: CVE(s) ID list
+      description: Enter a list of the CVE ID(s) associated with this query, one bullet for each distinct CVE. GitHub will automatically link CVE IDs to the [GitHub Advisory Database](https://github.com/advisories).
```
Comment (xcorail) [inline .github/ISSUE_TEMPLATE/bug-slayer.yml:22]:
```suggestion
      description: Enter a list of the CVE ID(s) associated with this query, one bullet for each distinct CVE. You need at least four high severity CVEs or two critical severity CVEs. 
```

**Comment by xcorail** [comment]
I messed up the history

**Pull request closed without merging**